news/2015-11-25-arq-container-se-CR1.asciidoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: news
+title: Arquillian container SE 1.0.0.CR1 released!
+author: Tomas Remes
+priority: 1
+change_frequency: weekly
+---
+
+We are pleased to announce new release of Arquillian container SE. If you are looking for reliable way to test your application in Java SE environment
```

Comment:
Also I would add a sentence or two about the reason for the creation, i.e. why it was needed and that there is no such container available...